plugins/ipam/dhcp/lease.go

```diff
@@ -67,6 +67,7 @@ type DHCPLease struct {
 	broadcast     bool
 	stopping      uint32
 	stop          chan struct{}
+	check         chan bool
```

Comment:
nit: it's convention to use the type `chan struct{}` to indicate that a channel carries no information.